radio/src/telemetry/multi.cpp

```diff
@@ -23,6 +23,11 @@
 
 #define MULTI_CHAN_BITS 11
 
+#if defined(LUA)
+  #define MULTI_BUFFER_SIZE 177
```

Comment:
it seems it is defined 2 times?
can it be dynamically allocated?

Reply:
Corrected and ynamic allocation implemented